app/src/main/res/layout/fragment_contacts.xml

```diff
@@ -11,14 +11,33 @@
         android:layout_height="wrap_content"
         app:title="@string/title_contacts" />
 
-    <com.yterletskyi.happyfriend.common.view.SearchView
-        android:id="@+id/etSearch"
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginHorizontal="8dp"
-        android:layout_marginVertical="8dp"
-        android:hint="@string/search_hint_contacts"
-        android:inputType="text" />
+        android:layout_height="wrap_content">
+        <com.yterletskyi.happyfriend.common.view.SearchView
+            android:id="@+id/etSearch"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="8dp"
+            android:layout_marginVertical="8dp"
+            android:hint="@string/search_hint_contacts"
+            android:inputType="text"
+            android:maxLength="35"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent"/>
+
+        <ImageButton
+            android:id="@+id/etClear"
+            android:layout_width="24dp"
+            android:background="@drawable/ic_baseline_close_24"
+            android:layout_height="24dp"
```

Comment:
Let's use `wrap_content` for width and height. It is already `24dp` by `24dp` in `ic_baseline_close_24.xml`